package.json

```diff
@@ -46,18 +46,20 @@
     "path-to-regexp": "^6.1.0",
     "qs": "^6.9.4",
     "query-string": "^6.13.1",
-    "react": "^16.8.6",
+    "react": "^17.0.2",
```

Comment:
There are some breaking changes, could you check them? Event listeners likely need update. https://reactjs.org/blog/2020/10/20/react-v17.html#changes-to-event-delegation

Reply:
It was for trial. Reverted back. I thought that may be it has SSRSuspense available..